src/pkgdev/mangle.py

```diff
@@ -85,6 +85,17 @@ def keywords_sort_key(kw):
                 break
         return change.update('\n'.join(lines) + '\n')
 
+    @mangle('metadata.xml')
+    def _metadata(self, change):
+        """Format metadata.xml file."""
+        if not change.path.endswith('/metadata.xml'):
+            return change
+        lines = change.data.splitlines()
+        from lxml import etree
```

Comment:
Imports generally shouldn't be embedded in functions except for specific reasons (e.g. optional support).

Also, don't forget to explicitly add `lxml` to requirements/dev.txt and requirements/install.txt. The tests only pass because it's already implicitly pulled in by pkgcore/pkgcheck.